pkg/ccl/changefeedccl/sink_kafka_v2.go

```diff
@@ -97,14 +99,7 @@ func newKafkaSinkClientV2(
 		kgo.ProducerOnDataLossDetected(func(topic string, part int32) {
 			log.Errorf(ctx, `kafka sink detected data loss for topic %s partition %d`, redact.SafeString(topic), redact.SafeInt(part))
 		}),
-	}
-
-	recordResize := func(numRecords int64) {}
-	if m := mb(requiresResourceAccounting); m != nil { // `m` can be nil in tests.
```

Comment:
removed this because it didnt work quite right, since the builder function returns `(*sliMetrics)(nil)` as a `metricsRecorder`, which is != nil here.